Add OpenSky Network API and remove Salesfly PDF API

arthurlli opened the pull request:
<!-- Thank you for taking the time to work on a Pull Request for this project! -->
<!-- To ensure your PR is dealt with swiftly please check the following: -->
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

arthurlli commented:
I found url for Salesfly PDF API is invalid, so I modified and attached another link, which is squashed with my previous commit. Hopes this helps.

matheusfelipeog commented on README.md:
```diff
@@ -377,7 +377,7 @@ API | Description | Auth | HTTPS | CORS |
 | [PrexView](https://prexview.com) | Data from XML or JSON to PDF, HTML or Image | `apiKey` | Yes | Unknown |
 | [Restpack](https://restpack.io/) | Provides screenshot, HTML to PDF and content extraction APIs | `apiKey` | Yes | Unknown |
-| [Salesfly PDF](https://salesflypdf.com) | HTML/URL to PDF | `apiKey` | Yes | Yes |
+| [Salesfly PDF](https://developers.salesfly.com/swift/) | HTML/URL to PDF | `apiKey` | Yes | Yes |
```
This API is in private beta.


![image](https://user-images.githubusercontent.com/50463866/123019699-5e4c7400-d3a7-11eb-8e64-981521bf9978.png)

Can you remove, please?

```suggestion
```

matheusfelipeog reviewed (changes requested):
Hi @arthurlli ✋ 

Thanks for your contribution, we are grateful for that ❤️ 

Can you resolve the review, please?

arthurlli commented:
Hi @matheusfelipeog, thank you for the review, I have updated and squashed the commits, please check. Thanks!:smile:

arthurlli commented:
> 
> 
> Hi @matheusfelipeog, thank you for the review, I have updated and squashed the commits, please check. Thanks!😄

I just noticed that I "force-push" the commit, sorry if it may cause some issue.

matheusfelipeog reviewed (approved):
Perfect ❤️

matheusfelipeog commented:
> I just noticed that I "force-push" the commit, sorry if it may cause some issue.

No problem, you did the right. 😆

arthurlli commented:
Great! Thank you!

matias904 reviewed (changes requested):
- [ ]
